src/app/shared/team-radio-switch/team-radio-switch.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="team-body">
+  <label for="team1" [ngClass]="refInput2.checked ? 'unchecked' : ''">
+    <input
+      id="team1"
+      type="radio"
+      [value]="team1.id"
+      [(ngModel)]="valueId"
+      #refInput1
+    />
+    <img [src]="team1.iconUrl" [alt]="team1.name" />
+    <div class="input-name">{{ team1.name }}</div>
+  </label>
+  <div class="vs">vs</div>
+  <label for="team2" [ngClass]="refInput1.checked ? 'unchecked' : ''">
```

Comment:
To avoid using of an empty else value you may use [ngClass]="{'unchecked': refInput1.checked}"

Reply:
done, thanks